tests/users.spec.ts

```diff
@@ -0,0 +1,118 @@
+import {
+    describe, test, expect
+} from '@jest/globals';
+import {UsersService} from "./services/userService.ts";
+import {generateUserData} from "./helpers/user-data-generator.ts";
+import {IUser} from "./types.ts";
+import {UserRepository} from "./repositories/userRepository.ts";
+import {DBClient} from "./db-client/DBClient.ts";
+import {Client} from "ts-postgres";
+import {uuid} from "../src/utils.ts";
+
+const usersService = new UsersService();
+let client: Client;
+let userRepository: UserRepository;
+
+beforeAll(async () => {
+    client = await DBClient.createAndConnectClient();
+    userRepository = new UserRepository(client);
+});
+
+afterAll(async () => {
+    await client.end();
```

Comment:
We can move this code to global after all, right?

Reply:
Not exactly. I tried to do this and add client shutdown to the global tearDown, but spec file and tearDown have different contexts. So, it doesn't close created connection to the DB and Jest cannot finish it's work saying 
```
Jest did not exit one second after the test run has completed.
'This usually means that there are asynchronous operations that weren't stopped in your tests.
```
And jest makes assumption that the problem is in totally different place when using `--detectOpenHandles`.  But debugging revealed the problem with different contexts. globalSetup and tearDown have the same context, but we will not be able to reach this connection from the spec file
So, we need to leave it where it is.